Sources/Storage/StorageActor.swift

```diff
@@ -0,0 +1,4 @@
+/// A global actor responsible for managing access and write operations in storage systems
+@globalActor public actor StorageActor {
```

Comment:
Check if this works:
```suggestion
@globalActor
public actor StorageActor {
```